client/dashboard/domains/domain-connection-setup/verification-in-progress-next-steps.tsx

```diff
@@ -0,0 +1,57 @@
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { rotateRight, globe, published } from '@wordpress/icons';
+import { ActionList } from '../../components/action-list';
+import { CollapsibleCard } from '../../components/collapsible-card';
+import { SectionHeader } from '../../components/section-header';
+
+import './style.scss';
+
+interface DomainConnectionNextStep {
+	id: string;
+	title: string;
+	description: string;
+	icon: JSX.Element;
+}
+
+export default function VerificationInProgressNextSteps() {
+	const data: DomainConnectionNextStep[] = [
+		{
+			id: 'automatic-verification',
+			title: __( 'Automatic verification' ),
+			description: __( 'We’ll continue checking your name servers until they’re all verified.' ),
+			icon: rotateRight,
+		},
+		{
+			id: 'global-propagation',
+			title: __( 'Global propagation' ),
+			description: __(
+				'Once name servers are verified, your domain name will gradually become live globally.'
+			),
+			icon: globe,
+		},
+		{
+			id: 'cache-propagation',
+			title: __( 'We’ll notify you when it’s ready' ),
+			description: __( 'No need to refresh this page. We’ll email you as soon as it’s done.' ),
+			icon: published,
+		},
+	];
+
+	return (
+		<CollapsibleCard
+			header={ <SectionHeader level={ 3 } title={ __( 'What happens next' ) } /> }
+			className="verification-in-progress-next-steps"
+		>
+			{ data.map( ( item ) => (
+				<ActionList.ActionItem
+					key={ item.id }
+					title={ item.title }
+					description={ item.description }
+					decoration={ <Icon icon={ item.icon } /> }
+					actions={ null }
```

Comment:
It feels weird to pass `null` to this component; an action list without action. This will also render an empty `<HStack>` I beleive. 

I see that the action items list does match the [designs](https://www.figma.com/design/8kmwTc6HM4GEWS4gvsuD84/CIAB-Plans--Billing--Upgrade---Domains?node-id=2113-37483&m=dev). 

It seems to me like abstracting the list part into a list component that can be used directly is a better path forward.

cc @lucasmendes-design @jasmussen

Reply:
That's was my original idea, but I found this discussion in Slack: p1756407573677299-slack-CNGQYA3B9
Happy to iterate and create a new componente if needed :)

Reply:
@StevenDufresne I'll deploy this PR and leave a note to replace the component once we have the new one 👍

Reply:
Can you open a linear issue, adding this context?